plugins/out_oracle_log_analytics/oci_logan.c

```diff
@@ -620,16 +658,372 @@ static flb_sds_t compose_uri(struct flb_oci_logan *ctx,
     flb_sds_cat_safe(&full_uri, ctx->uri, flb_sds_len(ctx->uri));
     flb_sds_cat_safe(&full_uri, "?", 1);
     flb_sds_cat_safe(&full_uri, uri_param, flb_sds_len(uri_param));
-
     flb_sds_destroy(uri_param);
 
     return full_uri;
 }
 
+// Checks if the current security token needs refresh(expires within 5min)
```

Comment:
use /* */ for single line https://github.com/fluent/fluent-bit/blob/master/CONTRIBUTING.md#single-line-comment